source/Mortgage.h

```diff
@@ -36,7 +36,7 @@ class Mortgage {
 	Mortgage() = default;
 	// Create a new mortgage of the given amount. If this is a fine, set the
 	// credit score to zero for a higher interest rate.
-	Mortgage(int64_t principal, int creditScore, int term = 365);
+	Mortgage(int64_t principal, int creditScore, int term = 365, const std::string &type="");
```

Comment:
Will this ever be called without a type? If not, then do not allow defaulting the type.

Reply:
Yes. The existing code needs a default for term and type.